handlers.go

```diff
@@ -17,8 +18,46 @@ func newSpecialReportsHandler(service specialReportService) specialReportsHandle
 	return specialReportsHandler{service: service}
 }
 
+func (h *specialReportsHandler) count(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	_, err := writer.Write([]byte(strconv.Itoa(len(ids))))
+	if err != nil {
+		log.Warnf("Couldn't write count to HTTP response. count=%d %v\n", len(ids), err)
+		writer.WriteHeader(http.StatusInternalServerError)
+	}
+}
+
+func (h *specialReportsHandler) ids(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	writer.Header().Add("Content-Type", "text/plain")
+	if len(ids) == 0 {
+		writer.WriteHeader(http.StatusOK)
```

Comment:
I also found that the design of concept loading could be enhanced a lot. That's true. I found lots of copy paste and other quickly written code. Not just in this app, but in the other transformers and concept-publisher itself.

Here's an example of another transformer, how it returns ids. It's a list of json entries but not wrapped in a []. 
https://github.com/Financial-Times/curated-authors-transformer#ids

concept-publisher also gets the count.
https://github.com/Financial-Times/concept-publisher/blob/master/service.go#L121

concept-publisher works this way with __ids
https://github.com/Financial-Times/concept-publisher/blob/master/service.go#L184
check line 202, what happens if it's a 404

My task is very high priority so i'm going to make the least changes to fix production.